dev_dep_versions.yml

```diff
@@ -1,3 +1,3 @@
-__version__: "2.4.0.dev0"
-__cuda_version__: "12.1"
-__tensorrt_version__: "10.0.1"
\ No newline at end of file
+__version__: "2.5.0.dev0"
+__cuda_version__: "12.4"
+__tensorrt_version__: "10.0.1"
```

Comment:
Should the version here be increased to `2.5.0` and should the `version.txt` be modified similarly?
https://github.com/pytorch/TensorRT/blob/643fb7b20dcac74c27da428e8bb3e631a906be25/version.txt#L1
cc: @narendasan